src/pages/HostingPreferences.vue

```diff
@@ -21,24 +21,33 @@ import PrimaryLayout from 'components/PrimaryLayout.vue'
 import HAppSelectionSection from 'components/settings/hostingPreferences/HAppSelectionSection'
 import InvoicesSection from 'components/settings/hostingPreferences/InvoicesSection.vue'
 import PricesSection from 'components/settings/hostingPreferences/PricesSection.vue'
+import { usePreferencesStore } from '@/store/preferences'
 import { reactive } from 'vue'
 
+const preferencesStore = usePreferencesStore()
+
 const invoicesSettings = reactive({
   frequency: {
     amount: 100,
     period: 7
   },
   due: {
     period: 7
-  }
+  },
+  ...rawInvoicesSettings
 })
 
 const pricesSettings = reactive({
   cpu: 100,
   storage: 100,
-  bandwidth: 100
+  bandwidth: 100,
+  ...rawPricesSettings
 })
 
+const rawPricesSettings = computed(() => preferencesStore.pricesSettings)
+
+const rawInvoicesSettings = computed(() => preferencesStore.invoicesSettings)
+
```

Comment:
@mateuszRybczonek  - This part is more a demonstration than a hard implementation.  It's here to underline the point that we prob need to keep some local state reference making the call to allow for the user to play with/locally update the prices before submitting the call to set the host invoice/price preferences - as this can only be done once.